src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/RemoteTypes/RemoteSourceRepository.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.VisualStudio.Shell;
+using Newtonsoft.Json;
+using NuGet.Configuration;
+using NuGet.Protocol.Core.Types;
+
+namespace NuGet.PackageManagement.VisualStudio.RemoteTypes
+{
+    public class RemoteSourceRepository : SourceRepository
```

Comment:
Is there a way you can avoid splitting remote vs. local scenarios? We want them to be at parity (sooner or later), so splitting them artificially may be creating debt for yourself.